packages/amplify-e2e-core/src/categories/auth.ts

```diff
@@ -540,6 +545,62 @@ export function addAuthWithSignInSignOutUrl(cwd: string, settings: any): Promise
   });
 }
 
+export function addAuthWithDefaultSocial_v4_30(cwd: string, settings: any): Promise<void> {
+  return new Promise((resolve, reject) => {
+    const { FACEBOOK_APP_ID, FACEBOOK_APP_SECRET, GOOGLE_APP_ID, GOOGLE_APP_SECRET, AMAZON_APP_ID, AMAZON_APP_SECRET } = getSocialProviders(
+      true,
+    );
+
+    spawn(getCLIPath(), ['add', 'auth'], { cwd, stripColors: true })
+      .wait('Do you want to use the default authentication and security configuration?')
+      .send(KEY_DOWN_ARROW)
+      .sendCarriageReturn()
+      .wait('How do you want users to be able to sign in?')
+      .sendCarriageReturn()
+      .wait('Do you want to configure advanced settings?')
+      .sendCarriageReturn()
+      .wait('What domain name prefix do you want to use?')
+      .sendCarriageReturn()
+      .wait('Enter your redirect signin URI:')
+      .sendLine('https://www.google.com/')
+      .wait('Do you want to add another redirect signin URI')
+      .sendLine('n')
```

Comment:
Can you use `sendConfirmNo()` in place of this here and elsewhere.

Reply:
sendConfirmedNo is used in this file exclusively now.